utils/post_utils.jsx

```diff
@@ -124,7 +124,7 @@ export function containsAtMention(text, key) {
     }
 
     // This doesn't work for at mentions containing periods or hyphens
-    return new RegExp(`\\B${key}\\b`, 'i').test(removeCode(text));
+    return new RegExp(`^(?!/).*\\B${key}\\b`, 'i').test(removeCode(text));
```

Comment:
Instead of changing the regex, could you change this to check if `text.startsWith('/')` above and return false if that is the case? That would make it a bit easier to follow for someone unfamiliar with regular expressions

Reply:
sure sounds good to me